tests/utils.go

```diff
@@ -2795,6 +2795,46 @@ func NewConsoleExpecter(virtCli kubecli.KubevirtClient, vmi *v1.VirtualMachineIn
 	}, timeout, opts...)
 }
 
+// The method adds the expect.BSnd command to the exect.BExp expression.
+// It is done to make sure the match was found in the result of the expect.BSnd
+// command and not in a leftover the wasn't removed from the buffer.
+func ExpectBatchWithValidatedSend(expecter expect.Expecter, batch []expect.Batcher, timeout time.Duration) ([]expect.BatchRes, error) {
+	sendFlag := 0
+	expectFlag := 0
+	previousSend := ""
+	for i, batcher := range batch {
+		switch batcher.Cmd() {
+		case expect.BatchExpect:
+			if expectFlag == 1 {
+				return nil, fmt.Errorf("Two sequential expect.BExp are not allowed")
+			}
+			expectFlag = 1
+			sendFlag = 0
+			bExp, _ := batch[i].(*expect.BExp)
+			previousSend := regexp.QuoteMeta(previousSend)
+
+			// Remove the \n since it is translated by the console to \r\n.
+			previousSend = strings.TrimSuffix(previousSend, "\n")
+			bExp.R = fmt.Sprintf("(?s)%s%s%s", previousSend, ".*", bExp.R)
```

Comment:
maybe the (?s) and the reason for it worth comment (let . match \n)

Reply:
Well, I don't think so. If a developer doesn't understand a regex, he/she can go to the regex [documentation](https://golang.org/pkg/regexp/syntax/). There is no need to duplicate it here.

Reply:
reason was as its a fundamental match of this method, that is eating the \n, but up to you.
btw , maybe it would be stronger to enforce that exactly one \n can come in between, this way no other stderr can occur in the middle ?

Reply:
I've update the regex so the `(?s`) will affect only the `.*` and not the whole expression.

Reply:
it still allow whatever test in between the previousSend and the bExp.R isnt it ?
dont we want more strict ?

Reply:
> reason was as its a fundamental match of this method, that is eating the \n, but up to you.

It is not related the the `\n` we trim. It just allows the output of the sent command to contain and value including `\n`.

> btw , maybe it would be stronger to enforce that exactly one \n can come in between, this way no other stderr can occur in the middle ?
The output of the sent command can be any sequence of characters. Why should we restrict it?

Reply:
> > reason was as its a fundamental match of this method, that is eating the \n, but up to you.
> 
> It is not related the the `\n` we trim. It just allows the output of the sent command to contain and value including `\n`.
> 

didnt meant otherwise, i just afraid if anything can be in between the two halfs, and then combined with other changes it can create new problems, but up to you, its a long shot that we can take care of in the future, unless you want already to make it stricter, which is good imo.

> > btw , maybe it would be stronger to enforce that exactly one \n can come in between, this way no other stderr can occur in the middle ?
> > The output of the sent command can be any sequence of characters. Why should we restrict it?

Reply:
How can I make it stricter?

Reply:
thinking out loud with you,
expect exactly previousSend + \n (or \r\n) + bExp.R 
without allowing wild card .* in between that can match anything (if i am not wrong) ?

Reply:
That won't work since the expected value in most of the cases doesn't include the whole output of the command.

Reply:
so imo , in the future, bExp.R should be a regex matching the expected result, instead having here a wildcard which can consume all the in between expressions.